.gitignore

```diff
@@ -68,3 +68,5 @@ tests/examples/openmp/*.mod
 tests/examples/openmp/*.o
 
 benchmarks/tmp/*
+
+my_main.py
```

Comment:
I don't think it is a good idea to add files to this folder following the convention that each individual developer uses for their test files. I recommend running tests in a separate folder if you are worried about accidentally committing your tests.